boards/arm/arduino_nano_33_ble/Kconfig.board

```diff
@@ -5,16 +5,6 @@ config BOARD_ARDUINO_NANO_33_BLE
 	bool "Arduino Nano 33 BLE board"
 	depends on SOC_NRF52840_QIAA
 
-config BOARD_ARDUINO_NANO_33_BLE_EN_USB_CONSOLE
```

Comment:
so all boards cleaned up in this commit 9c034972b1e618bfddc7365360ba10bd924e9e54 must now use `-DSHIELD=cdc_acm_console`

That sounds like a step backward in user-friendliness if those boards are expected to always use the usb as console.

For example, this board `bl654_usb` doesn't seem to have any other option than using USB for console:
https://docs.zephyrproject.org/latest/boards/arm/bl654_usb/doc/bl654_usb.html

@kieran-mackey FYI.

Reply:
No, it is a step forward, only with these patches it becomes possible to use CDC ACM UART (_not usb_) as backend for console. The configuration of this board in the tree is not usable.

Reply:
>  The configuration of this board in the tree is not usable.

Which board or all boards ?

Read my comment:
> so all boards cleaned up in this commit 9c03497 must now use -DSHIELD=cdc_acm_console

i'm refering to the entire commit 9c03497, not a single board, but github requires me to put the comment on a single line.
That is, my comment covers all those boards you are modifying in that commit:
- arduino_nano_33_ble
- bl654_usb
- degu_evk
- nrf52840dongle_nrf52840

are you saying that the current configuration in Zephyr of those boards are not working for any of them ?

Reply:
as far as I'm aware, the general problem started with this cleanup:
https://github.com/zephyrproject-rtos/zephyr/commit/d12331ea4b5d8660f50f1dde007e1441849bc361
which again was fixing:
https://github.com/zephyrproject-rtos/zephyr/issues/38403
caused by this:
https://github.com/zephyrproject-rtos/zephyr/pull/37512

So maybe we should go back to the origin of the problem, instead of completely changing how users must use the boards.